weather/temperature_test.go

```diff
@@ -0,0 +1,50 @@
+package weather
+
+import (
+	"testing"
+
+	"github.com/jarcoal/httpmock"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTemperature(t *testing.T) {
+	test_city := "testville"
+	api_key := "apikey"
+
+	httpmock.Activate()
+	defer httpmock.DeactivateAndReset()
+
+	httpmock.RegisterResponder("GET", locationURL+api_key+"&q="+test_city,
```

Comment:
This is neat. It's different than how we use httpmock. I think one drawback of this approach is that you cannot make assertions about the contents of the http request.